wercker-step.yml

```diff
@@ -1,5 +1,5 @@
 name: validate-version
-version: 0.0.9
+version: 1.0.0
```

Comment:
This will resolve the issue of `latest` landing on `0.0.71`. For some reason we had a trailing `1` on that version and so it's always our latest `0.0.x` release right now - forcing us to set a fixed version in the products `wercker.yml`. No bueno 🌮